src/tests/ftest/security/cont_owner.py

```diff
@@ -0,0 +1,127 @@
+"""
+  (C) Copyright 2024 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import os
+
+import security_test_base as secTestBase
+from cont_security_test_base import ContSecurityTestBase
+from pool_security_test_base import PoolSecurityTestBase
+
+
+class DaosContainerOwnerTest(ContSecurityTestBase, PoolSecurityTestBase):
+    # pylint: disable=too-few-public-methods,too-many-ancestors
+    """Test daos_container user acls.
+
+    :avocado: recursive
+    """
+
+    def _create_cont_with_acl(self, cont_type):
+        # Set up an ACL that will allow us to reclaim the container
+        acl_file_name = os.path.join(self.tmp, "cont_test_owner_acl.txt")
+        acl_entries = [
+            secTestBase.acl_entry("user", self.current_user, "rwdaAtTo"),
+        ]
+        secTestBase.create_acl_file(acl_file_name, acl_entries)
+
+        # Set up the pool and container.
+        self.add_pool()
+        self.container = self.create_container_with_daos(self.pool, acl_file=acl_file_name,
+                                                         cont_type=cont_type)
+
+    def test_container_set_owner_no_check_non_posix(self):
+        """
+        Description:
+            Verify that daos container set-owner --no-check flag ignores missing user for non-POSIX
+            container.
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=vm
+        :avocado: tags=security,container,cont_sec,cont_set_owner
+        :avocado: tags=DaosContainerOwnerTest,test_container_set_owner_no_check
+        :avocado: tags=test_container_set_owner_no_check_non_posix
```

Comment:
Tags starting with `test_` should have a 1:1 mapping with the test method name. We shouldn't use `test_` tags for  larger groupings
I plan to add a lint check for this, and can handle this when I do, if this PR is otherwise ready
```suggestion
        :avocado: tags=DaosContainerOwnerTest,test_container_set_owner_no_check_non_posix
```

Reply:
I can fix this since I'm making more test fixes below.

Reply:
Fixed